.jenkins/sarus_run_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+set -x
+
+# Set up the compute node environment
+module load daint-gpu
+module add /project/d107/install/modulefiles/
+module load cray-python gcloud sarus
+gcloud auth configure-docker
+
+
+# Set up the working directing for the c48 test
+cd examples
+. /project/d107/install/venv/sn_1.0/bin/activate
+python create_arbitrary_rundir.py c48_config.yml "./c48_test"
+deactivate
+
+cd ${PWD}/c48_test
+cp ../../.jenkins/job_jenkins_sarus .
+export SCRATCH_DIR=${PWD}
+
+# Import the Docker images to be tested on Daint
+dockerfiles=( $( ls ../../docker/Dockerfile.* ) )
+for df in ${dockerfiles[@]}; do
+   arch=${df:24}
+   export FV3_CONTAINER=fv3gfs-compiled:$arch
+   export TAR_FILE=${FV3_CONTAINER/":"/"_"}.tar
+   gsutil copy gs://vcm-ml-public/jenkins-tmp/${TAR_FILE}.gz .
```

Comment:
!!! I just realized, this is copying what is currently private code to a public google cloud bucket. This needs to be fixed (even if it won't be a problem in the near future).

If you switch to using docker only to push/pull the images and `docker save` when you need to make a tar for sarus to load, this can be avoided.

Reply:
I guess we could delete the tar file in Google Storage after we try running the imported image via Sarus.  We would mean rebuilding the images every time we want to test them on Daint and defeats the purpose of having the build and runs tests separate.

Reply:
@ofuhrer what's the timeline on being able to make these images public?

Reply:
We were pushing to a public bucket that we could have access to from daint, but now we have a way to authenticate on piz daint (https://paper.dropbox.com/doc/Piz-Daint-Help--A5YJSvGiim4gABCtejPwJrCDAg-9xJgowSv9GUC2O6tS1L4g), and Oli has done so for the user olifu, I think it might be possible to use a private bucket

Reply:
I verified in the fv3core-daint-and-cloud jenkins plan that daint can read from our private buckets, so long as olifu is authenticated against google cloud, so we shouldn't have to push these to public buckets
https://jenkins.ginko.ch/job/fv3core-daint-and-cloud/34/

Reply:
OK then, let's please use these private buckets.

Reply:
Have switched to the dedicated Google Storage buckets for Jenkins/CI use